README.md

```diff
@@ -20,7 +22,7 @@ lazy val `hello-impl` = (project in file("hello-impl"))
       "com.lightbend.lagom" %% "lagom-javadsl-akka-discovery-service-locator" % "0.0.12",
 
       // for example, include Akka Discovery Kubernetes implementation
-      // see https://doc.akka.io/docs/akka/current/discovery/index.html
+      // see https://doc.akka.io/docs/akka/2.5/discovery/index.html
```

Comment:
Why this?
This will only create extra maintenance.

Reply:
Because it's more stable.  If, for example, Akka 2.6 moves the docs, this URL won't break.

Reply:
But this decision is based on an `if` that may not occur. What will occur is that will need now to update this to not point to outdated documentation. 

The same goes for the links here internally. On each release you need to update the README only because if something changes you don't get a broken link. Now, you can't simply go create a tag and cut a release. You need to update the README file as well.

Reply:
So let's separate the two changes.  When we upgrade to Akka 2.6, we'll update this README.

For the internal links, we should use relative links instead and avoid the whole problem: https://github.com/lagom/lagom-akka-discovery-service-locator/pull/53

But with regards to updating the README for a release, that's still a problem with:

```sbt
"com.lightbend.lagom" %% "lagom-javadsl-akka-discovery-service-locator" % "0.0.12"
```

and

```xml
<dependency>
    <groupId>com.lightbend.lagom</groupId>
    <artifactId>lagom-javadsl-akka-discovery-service-locator_2.12</artifactId>
    <version>0.0.12</version>
</dependency>
```

Reply:
Yep, that's true. 

Relative links is a better solution as well.